hawc/apps/epiv2/constants.py

```diff
@@ -63,6 +63,7 @@ class VarianceType(models.IntegerChoices):
 
 
 class ConfidenceIntervalType(models.TextChoices):
+    NONE = "NA", NO_LABEL
```

Comment:
There are a few fields like this where the scientists asked us to make them not required, but they already have an option that is "---" or "Not reported". Maybe the "---" is confusing and they think it isn't a valid option? or we should just hide the asterisk on those fields?

Reply:
Perhaps N/A would be a better display value for these fields?

Reply:
agree!